docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -580,6 +662,8 @@ You can also specify authentication credentials with the proxy host.
 testcafe chrome my-tests/**/*.js --proxy username:password@proxy.mycorp.com
 ```
 
+*Related configuration file property*: [proxy](configuration-file.md#proxy).
+
 ### --proxy-bypass \<rules\>
 
 Specifies the resources accessed bypassing the proxy server.
```

Comment:
accessed bypassing?

Reply:
Yes, it's "ing" and - do you mind "bypass" at all? Anyway, there seems to be no expression with "bypass" that doesn't involve "ing". So, what's the right synonym - I came across "in circumvention of" - is it okay?

*Specifies the resources accessed in circumvention of the proxy server.*

Reply:
No, *circumvent* was a very bad idea, the common word is *bypass*.

![image](https://user-images.githubusercontent.com/11459924/53736390-a4a9fc00-3e9a-11e9-8c24-ab077d8c5745.png)

Reply:
So, something like:

*Requires that TestCafe bypasses the proxy server to access the specified resources.*